idiotquant/__init__.py

```diff
@@ -0,0 +1 @@
+from .core import hmm
```

Comment:
hmm?

Reply:
this is other package method name in core py. check this out